classes/migration/install/SubmissionsMigration.php

```diff
@@ -175,6 +177,10 @@ public function up(): void
             $table->foreign('user_id', 'subeditor_submission_group_user_id')->references('user_id')->on('users')->onDelete('cascade');
             $table->index(['user_id'], 'subeditor_submission_group_user_id');
 
+            $table->bigInteger('user_group_id');
+            $table->foreign('user_group_id')->references('user_group_id')->on('user_groups');
```

Comment:
I'd recommend adding `onDelete('cascade')` here. Currently, if someone attempts to delete the user group without first clearing out this table, it'll cause an error.